_posts/en/newsletters/2019-11-06-newsletter.md

```diff
@@ -0,0 +1,152 @@
+
+    Discussion remains ongoing as solutions to the above concerns are
+    suggested and the proposal receives additional review.
+
+- **Signaling support for address relay:** full nodes share the IP
+  addresses of other full nodes they've heard about with their peers
+  using the P2P protocol's `addr` (address) message, enabling fully
+  decentralized peer discovery.  SPV clients can also use this mechanism
+  to learn about full nodes, although most clients currently use some
+  form of centralized peer discovery and so `addr` messages sent to those
+  clients are wasted bandwidth.
+
+    Gleb Naumenko sent an [email][naumenko addr relay] to the
+    Bitcoin-Dev mailing list suggesting that nodes and clients should
+    signal to their peers whether or not they want to participate in
+    address relay.  This will avoid wasting bandwidth on clients that
+    don't want the addresses and can make it easier to determine the
+    consequences of certain network behavior related to address relay.
+
+    Two methods are proposed for allowing nodes to indicate whether or
+    not they want to participate in address relay---a per-node method
+    and a per-connection method.  The per-node method could easily build
+    on top of work already being done for addrv2 messages (see [Newsletter
+    #37][news37 addrv2]), but it would be less flexible than the the
+    per-connection method.  In particular, the per-connection method
+    could allow a node to dedicate some connections to transaction relay
+    and other connections to address relay, producing possible privacy
+    advantages.  Naumenko's email requests feedback on which method would
+    be preferred by implementers of both full nodes and lightweight
+    clients.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #16943][] adds a `generatetodescriptor` RPC that allows
+  new blocks generated during testing (e.g. in regtest mode) to pay a
+  script represented by an [output script descriptor][].  Prior to this
+  change, only the `generatetoaddress` RPC was available and it can only
+  pay P2PKH, P2SH, or P2WSH addresses.
+
+- [C-Lightning #3220][] begins always creating signatures with a low *r*
+  value, reducing the maximum size of the signature by one byte and
+  saving about 0.125 vbytes on average per C-Lightning peer in onchain
+  transactions.  Bitcoin Core also previously adopted this change to its
+  wallet's signature-generation code (see [Newsletter #8][news8 lowr]).
+
+- [LND #3558][] synthesizes a unified policy for any case where two
+  particular nodes have multiple channels open between them and then
+  uses this synthetic policy when considering routing through any of
```

Comment:
nit: maybe just me, but "synthetic policy" here could be read as fake/ersatz, perhaps "unified policy" or "policy"?